packages/nc-gui/components/virtual-cell/Formula.vue

```diff
@@ -20,7 +20,7 @@ const showEditFormulaWarningMessage = () => {
   }, 3000)
 }
 
-const result = computed(() => (isPg.value ? handleTZ(value) : value))
+const result = computed(() => (isPg.value ? handleTZ(value?.value) : value?.value))
```

Comment:
Can we rename `value` to `cellValue` so it's not as confusing with `value.value` 😄

Reply:
Good idea.